cmd/modern/root/config/connection-strings.go

```diff
@@ -36,6 +38,13 @@ func (c *ConnectionStrings) DefineCommand(...cmdparser.CommandOptions) {
 	}
 
 	c.Cmd.DefineCommand(options)
+
+	c.AddFlag(cmdparser.FlagOptions{
+		String:        &c.database,
+		Name:          "database",
+		DefaultString: "master",
```

Comment:
wouldn't the default be empty so the user gets their default database?

Reply:
I think I need some clarification of the intent of this function. 
Pls let me know if my understanding is correct:

If the user creates a context in the config file, it could be in one of 2 states:
- no database set
- a specific database is set

When the user runs the ConnectionStrings command, if they don't pass -d, we should provide whatever database is in the context, which could mean not providing any specific database name. If they do pass -d, we would override the database name from the context.

Reply:
> if they don't pass -d, we should provide whatever database is in the context, which could mean not providing any specific database name. 

This would be the best option IMHO. By default, you get the database the has been set in the context. `-d` can override that

Reply:
Yeah, agree.  We don't actually set the database in the context, we set in on the login as DEFAULT_DATABASE.  So this would mean we need sqlcmd to connect to the container/sqlservr, get the DB_NAME() and use this by default (when -d is not passed in)